app/src/main/java/com/nextcloud/talk/controllers/util/ControllerViewBindingDelegate.kt

```diff
@@ -43,7 +43,7 @@ class ControllerViewBindingDelegate<T : ViewBinding>(
         })
     }
 
-    override fun getValue(thisRef: Controller, property: KProperty<*>): T {
-        return binding ?: viewBinder(thisRef.view!!).also { binding = it }
+    override fun getValue(thisRef: Controller, property: KProperty <*>): T? {
+        return binding ?: thisRef.view?.let { viewBinder(it).also { binding = it } }
```

Comment:
This looks a bit too complicated to still be a `?:`. I'd change this into a proper if-else for readability.

Reply:
you mean like in the last commit b37bd59ddc3a2d698180fcc2535a2531585b5199 ?

Reply:
simpler still, see comment below